highfidelity/hifi
# Bubble button blink fix

https://highfidelity.manuscript.com/f/cases/13034/Bubble-icon-blinks-three-times-when-clicked-on

Test plan
- Press Bubble button (if its active, press it to deactivate)
- make sure button does not blinking when activating
